lib/graphql/client.rb

```diff
@@ -44,6 +44,8 @@ class ValidationError < Error; end
     # This ability will eventually be removed in future versions.
     attr_accessor :allow_dynamic_queries
 
+    attr_accessor :raise_on_unknown_enum_values
```

Comment:
At the time of the check to raise, we have a single instance. I think the setting's name could reflect this.

```suggestion
    attr_accessor :raise_on_unknown_enum_value
```